fdbserver/MoveKeys.actor.cpp

```diff
@@ -83,6 +82,11 @@ ACTOR Future<MoveKeysLock> takeMoveKeysLock( Database cx, UID ddId ) {
 				lock.prevWrite = readVal.present() ? BinaryReader::fromStringRef<UID>(readVal.get(), Unversioned()) : UID();
 			}
 			lock.myOwner = deterministicRandom()->randomUniqueID();
+			TraceEvent("TakeMoveKeysLockTransaction", ddId)
```

Comment:
Moving the trace event after tr.commit can make the transaction duration smaller, I think.